### DISCUSSION TITLE
Devocub Antichatter

### DISCUSSION THREAD

**Discussion by yuikaze** [OpenTabletDriver/OpenTabletDriver#1540] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/92944521/138326158-9a3113bb-b6d0-499f-a7a7-bd0c458a020b.png)
When I enable Devocub's antichatter, does it also include the smoothing? I'm confused because I also tried enabling Hawku Smoothing Filter along with Devocub's antichatter, it's not compatible. 
Hope this gets answered because I might be using only Anti Chatter all along without smoothing after all.
Also do I necessarily need to use the prediction setting?

**Comment by jamesbt365** [comment]
If you want smoothing just use it separately using a different plugin, you shouldn't need to use anti chatter and smoothing at the same time.

I recommend bezierinterpolator or radial flow smoothing plugins if you want smoothing.

**Reply by yuikaze** [reply]
so this devocub anti chatter is different from what vaxei and whitecat is using from the actual devocub tablet driver that has smoothing + anti chatter?

**Reply by jamesbt365** [reply]
The code is exactly the same, however hawku and devocub and broken with there filters and apply too much smoothing and anti chatter. You shouldn't just copy different players areas and settings though, use what works for you.

But its the same, i dont use them so i couldn't tell you if smoothing is in them but im assuming so anyway, i've never used devocub but if smoothnig is auto applied there then its auto applied here as well.

**Comment by X9VoiD** [comment, marked as answer]
Devocub Antichatter is the smoothing itself, as can be seen by "Latency" and "Frequency" options, which are the only options for Hawku Smoothing (the algorithm that antichatter is based on).
